src/typescript/frontend/src/utils/geolocation.ts

```diff
@@ -0,0 +1,59 @@
+import { GEOBLOCKED, GEOBLOCKING_ENABLED, VPNAPI_IO_API_KEY } from "lib/server-env";
+
+export type Location = {
+  country: string;
+  region: string;
+  countryCode: string;
+  regionCode: string;
+  vpn: boolean;
+};
+
+const isAllowedLocation = (location: Location) => {
```

Comment:
```suggestion
const isNotAllowedLocation = (location: Location) => {

// or maybe

const isLocationGeoblocked
```

Let's make the phrasing of the boolean geoblocked logic functions to be consistent so that it's more natural to read, otherwise it gets a little confusing reading it.

I think this and `isBanned` are the only two we use. Also let's change `isBanned` to `isUserGeoblocked` just so to keep all the language consistent